glide.lock

```diff
@@ -169,4 +184,63 @@ imports:
   - pkg/util/yaml
   - pkg/watch
   - pkg/watch/versioned
+- name: k8s.io/kubernetes
```

Comment:
Yup :(
But this lets us add validations for the generated Kubernetes objects in kubernetes.go.
Currently, I am only doing this for secrets, but using this package we can validate all the generated artifacts.
This is referenced in #139, and this is how I am doing this for secrets -
```golang
	validationErrors := kvalidation.ValidateSecret(&kapi.Secret{
		ObjectMeta: kapi.ObjectMeta{
			Name: sec.ObjectMeta.Name,
			// Putting namespace as default here because ValidateSecret errors
			// out if no namespace is provided
			Namespace: "default",
		},
		Data: sec.Data,
	})
```